services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouterController.java

```diff
@@ -158,7 +194,41 @@ public RouteTableWebJson getOrCreateVpcRouteTable(@PathVariable String projectid
             RestPreconditionsUtil.verifyParameterNotNullorEmpty(projectid);
             RestPreconditionsUtil.verifyResourceFound(projectid);
 
-            routetable = this.routerService.getOrCreateVpcRouteTable(projectid, vpcid);
+            routetable = this.routerService.getVpcRouteTable(projectid, vpcid);
+
+        } catch (ParameterNullOrEmptyException e) {
+            throw e;
+        } catch (CanNotFindVpc e) {
+            logger.log(Level.WARNING, e.getMessage() + " : " + vpcid);
+            throw e;
```

Comment:
@DavidLiu506 Same here. Do we need to throw an exception for a `vpc not found` here?

Reply:
Talked with James. Throw vpc not found.

Reply:
agree